torch/csrc/jit/codegen/cuda/python_frontend/python_bindings.cpp

```diff
@@ -193,8 +193,11 @@ void initNvFuserPythonBindings(PyObject* module) {
           py::arg("dtype") = torch::jit::fuser::cuda::DataType::Float,
           py::return_value_policy::reference)
       .def(
+          // TODO: Should the inernals of this function live more explicitly in
```

Comment:
I don't have a known way to test this. @kevinstephano can you check this change and make sure it works please?

Reply:
The examples work with the change.